src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -0,0 +1,53 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.ExampleSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+/**
+ * An example command that uses an example subsystem.
+ */
+public class ShootCommand extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+
+  /**
+   * Creates a new ExampleCommand.
+   *
+   * @param subsystem The subsystem used by this command.
+   */
+  double maxRPM = 5700;
+  public ShootCommand() {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(RobotContainer.mShooterSubsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+      RobotContainer.mShooterSubsystem.setVelocitySetpoint(.9*maxRPM);
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+      RobotContainer.mShooterSubsystem.setVelocitySetpoint(0);
```

Comment:
Same thing as in drive base. This will use PID to brake the motor as fast as possible. We don't need this and could let the motor cost down to 0rpm. So do sparkMax.Set(0) instead which will set output to 0 percent.